backend/homework/7/src/main/java/org/example/services/VehicleServiceFactoryOne.java

```diff
@@ -0,0 +1,53 @@
+package org.example.services;
+
+import org.example.models.Speaker;
+import org.example.models.Tyre;
+import org.example.models.Vehicle;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Gives the inforamtion about the Vehicles generated by Factory one
+ */
+
+@Component
+@Qualifier("factory1")
```

Comment:
Use meaningful and descriptive names for your variables rather than using numerical suffixes.